ff/src/fields/models/mod.rs

```diff
@@ -18,12 +18,47 @@ use crate::{
 };
 use ark_serialize::*;
 
-impl_Fp!(Fp64, Fp64Parameters, BigInteger64, BigInteger64, 1);
-impl_Fp!(Fp256, Fp256Parameters, BigInteger256, BigInteger256, 4);
-impl_Fp!(Fp320, Fp320Parameters, BigInteger320, BigInteger320, 5);
-impl_Fp!(Fp384, Fp384Parameters, BigInteger384, BigInteger384, 6);
-impl_Fp!(Fp768, Fp768Parameters, BigInteger768, BigInteger768, 12);
-impl_Fp!(Fp832, Fp832Parameters, BigInteger832, BigInteger832, 13);
+impl_Fp!(Fp64, Fp64Parameters, BigInteger64, BigInteger64, 1, "64");
```

Comment:
why is there a new argument to all these macros

Reply:
The macro did change: https://github.com/arkworks-rs/algebra/pull/246/files#diff-f1aff030e059aa8177e8fe3e65c6c29a4fb0e0b693143f9e0962d1e61cfdd3e8

The reason for the new argument is so that we can mention the field size in the docs.

Reply:
Oh I see, I just missed that. Thanks!